win32ss/gdi/ntgdi/font.h

```diff
@@ -31,7 +31,7 @@ typedef struct _FONT_CACHE_ENTRY
     FT_BitmapGlyph BitmapGlyph;
     int Height;
     FT_Render_Mode RenderMode;
-    MATRIX mxWorldToDevice;
+    FT_Matrix mat;
```

Comment:
You can keep the previous name of this member so that we can know to what kind of matrix "mat" refers in the code.

Reply:
Naming should be consistent. `mx` stands for `MATRIX`. `mat` stands for `FT_Matrix`.

Reply:
sure (about the "mat" vs. "mx"); I was actually thinking about appending "WorldToDevice" in case it still makes sense.